.github/workflows/auto-merge-bot.yml

```
@@ -17,9 +17,9 @@ jobs:
         id: auto-merge-bot
         with:
           GITHUB-TOKEN: ${{ secrets.TOKEN }} 
-          CORE_EDITORS: "@MicahZoltu,@lightclient,@axic,@gcolvin,@SamWilsn,@Pandapip1"
+          CORE_EDITORS: "@lightclient,@axic,@gcolvin,@SamWilsn,@Pandapip1"
           ERC_EDITORS: "@lightclient,@axic,@SamWilsn,@Pandapip1"
-          NETWORKING_EDITORS: "@MicahZoltu,@lightclient,@axic,@SamWilsn"
+          NETWORKING_EDITORS: "@lightclient,@axic,@SamWilsn"
           INTERFACE_EDITORS: "@lightclient,@axic,@SamWilsn,@Pandapip1"
           META_EDITORS: "@lightclient,@axic,@gcolvin,@SamWilsn,@Pandapip1"
           INFORMATIONAL_EDITORS: "@lightclient,@axic,@gcolvin,@SamWilsn,@Pandapip1"
```

```suggestion
          INFORMATIONAL_EDITORS: "@MicahZoltu,@lightclient,@axic,@gcolvin,@SamWilsn,@Pandapip1"
```
I feel like this particular change is very important.